pkg/chunk/aws/s3_storage_client.go

```diff
@@ -47,6 +47,10 @@ func NewS3ObjectClient(cfg StorageConfig, schemaCfg chunk.SchemaConfig) (chunk.O
 		return nil, err
 	}
 
+	if cfg.S3ForcePathStyle {
+		s3Config = s3Config.WithS3ForcePathStyle(true)
```

Comment:
```suggestion
		s3Config = s3Config.WithS3ForcePathStyle(cfg.S3ForcePathStyle)
```

(and remove the `if`)

Reply:
looks better. thanks